semgrep.yaml

```
@@ -1,14 +1,14 @@
 package:
   name: semgrep
   version: 1.40.0
-  epoch: 0
+  epoch: 1
   description: "Lightweight static analysis for many languages. Find bug variants with patterns that look like source code."
   copyright:
     - license: LGPL-2.1
   dependencies:
     runtime:
       - python3
-      - py3-semgrep
+      - py3-semgrep=${{package.version}}
```

```suggestion
      - py3-semgrep
```

Melange doesn't support this yet, but at least having them co-versioned should make this less necessary.